.circleci/config.yml

```diff
@@ -628,3 +629,20 @@ workflows:
     jobs:
       - create-release-branch:
           e: default-executor
+
+  dependabot-automerge:
+    triggers:
+      - schedule:
+          # TODO: change schedule before merge - currently set for freq testing
+          cron: '10 * * * *'
+          # Every hour during regular working hours (9am-5pm) 
+          # on Tue to Fri as Monday is release freeze.
+          # cron: '0 9-17 * * 2-5'
```

Comment:
- 8pm to 6 am, wed-thurs
- check tz in circleci and adjust accordingly 
- start with 10 PRs per week ?